sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/DigitalTwinMetadataJsonConverter.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Azure.DigitalTwins.Core
+{
+#pragma warning disable AZC0014 // Avoid using banned types in public API
```

Comment:
Technically not part of the client API, but for BasicDigitalTwin, a helper class to deserialize unknown twin types and explore them. This helps make that experience less cumbersome.